src/main/java/seedu/address/logic/commands/AddTaskCommand.java

```diff
@@ -21,13 +20,13 @@ public class AddTaskCommand extends Command {
     public static final String COMMAND_WORD = "addTask";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Adds a task to the person identified by the index number in the displayed address book.\n"
-            + " Parameters: INDEX_OF_PERSON (must be positive integers) "
+            + ": Adds a task to the student identified by the index number in the displayed address book.\n"
```

Comment:
Maybe instead of address book, can change it to student list?

`Adds a task to the student identified by the index number in the displayed student list`

Reply:
Okie